Sources/ViewInspector/InspectionEmissary.swift

```diff
@@ -268,8 +262,10 @@ private extension Inspector {
         let expectation = XCTestExpectation(description: description)
         value[keyPath: keyPath] = { body in
             inspection(body)
-            ViewHosting.expel(function: function)
-            expectation.fulfill()
+            Task { @MainActor in
+                ViewHosting.expel(function: function)
```

Comment:
I'm thinking the inspection should happen in the task as well, so as to align with previous behavior.